### PR TITLE
Fix websockets process after celery upgrade

### PR DESCRIPTION
**Pull request by dannyroberts**
## Technical Summary
https://dimagi-dev.atlassian.net/browse/SAAS-13854

### Safety story
Right now the websockets feature is entirely broken, so this could only improve it or keep it the same.

### Automated test coverage

One major reason the bug was introduced in the first place is that there is nothing testing the uwsgi bootstrapping process. If our tests ran `deployment/websocket_wsgi.py` at all, it would have been obvious it wasn't starting up properly.

In this PR I have added those tests. Amit also pointed out that he stumbled on the same issue for gunicorn on staging initially, which uses `deployement/gunicorn/commcarehq_wsgi.py`. There's also `corehq/celery.py` that I found that does similar bootstrapping. So I added a test that executes each of those entrypoint modules (without actually starting a daemon process) and will fail if there are any errors.

### QA Plan

I'll put this on staging and just make sure it works.

### Migrations

- [x] The migrations in this code can be safely applied first independently of the code

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

(the only consideration is that it would undo this fix)

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by AmitPhulera — APPROVED** [review]
I am hopeful that this should fix it 🤞 
We did the same thing in `commcarehq_wsgi.py` to fix the startup issues when we deployed on staging.

**Comment by dannyroberts** [comment]
I see. Okay I'm going to modify `commcarehq_wsgi.py` to use the same helper functions for consistency.

**Review by biyeun — APPROVED** [review]

**Review by millerdev — APPROVED** [review]

**Comment by millerdev** [comment]
Thanks for adding these tests!

File: scripts/test-prod-entrypoints.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+set -e
+
+echo "Testing gunciron entrypoint"
+python -c 'import deployment.gunicorn.commcarehq_wsgi'
+
+echo "Testing websockets entrypoint"
+uwsgi --http-socket web.socket --test deployment.websocket_wsgi
+
+echo "Testing celery entrypoint"
+python -c 'import corehq.celery'
```
[the remark this answers is not in the document]
Comment (dannyroberts) [reply scripts/test-prod-entrypoints.sh:11]: Yeah that's right. And each of these fails if the bootstrapping isn't done correctly, for example if I comment out `run_patches()`.

**Pull request merged**